src/components/Input.tsx

```diff
@@ -29,6 +30,22 @@ export interface InputProps extends Omit<TextInputProps, 'style'> {
   renderCustomRightView?: () => ReactNode;
 }
 
+export interface LabelProps extends Omit<TextInputProps, 'style'> {
+  label?: string;
+  labelStyle?: StyleProp<TextStyle>;
+}
+
+export const Label = ({ label, labelStyle }: LabelProps) => (
+  <Text
+    style={[
+      fonts.regular.medium,
+      { color: colors.inkDarkest, marginBottom: spacing.sm },
```

Comment:
too many inline styles

Reply:
no, I just move it, I did not write this code. I just did export the label